test-server/fixtures/pages/iframe-same-origin.html

```diff
@@ -0,0 +1,6 @@
+<!doctype html>
+<html>
+  <body>
+    <iframe title="the-iframe" src="/iframe-content" width="700px" height="500px"></iframe>
+  </body>
```

Comment:
Nitpick -- should the same width and height be on the cross-origin iframe as well?